android/src/main/cpp/NativeProxy.cpp

```diff
@@ -370,6 +372,12 @@ void NativeProxy::installJSIBindings(
             tag);
       });
 
+  layoutAnimations->cthis()->setFindSiblingForSharedView([weakModule](int tag) {
+    return weakModule.lock()
+        ->layoutAnimationsManager()
+        .findSiblingForSharedView(tag);
```

Comment:
`weakModule.lock()` may return `nullptr`
```suggestion
    if (auto module = weakModule.lock()) {
      return module->layoutAnimationsManager().findSiblingForSharedView(tag);
    } else {
      return -1;
    }
```